1.0.1 release results in segfault on M1 mac

I tried installing lefthook on a new m1 mac and ran into segfaults on every command.

```
 λ  lefthook run pre-commit
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x2 addr=0x160 pc=0x10071d2c0]

goroutine 1 [running]:
github.com/spf13/viper.(*Viper).AllKeys(0x0)
	/home/ian/go/pkg/mod/github.com/spf13/viper@v1.9.0/viper.go:1942 +0x60
github.com/spf13/viper.(*Viper).AllSettings(0x0)
	/home/ian/go/pkg/mod/github.com/spf13/viper@v1.9.0/viper.go:2023 +0x34
github.com/spf13/viper.(*Viper).Unmarshal(0x0, {0x1007ef9a0, 0x1400022a4c0}, {0x0, 0x0, 0x0})
	/home/ian/go/pkg/mod/github.com/spf13/viper@v1.9.0/viper.go:1038 +0x28
github.com/evilmartians/lefthook/internal/config.mergeCommands(0x140000f3680, 0x140000f3800)
	/home/ian/src/evil/lefthook/internal/config/command.go:75 +0x2b8
github.com/evilmartians/lefthook/internal/config.unmarshalHooks(0x140000f3680, 0x140000f3800)
	/home/ian/src/evil/lefthook/internal/config/hook.go:47 +0x40
github.com/evilmartians/lefthook/internal/config.addHook({0x10075cddb, 0xa}, 0x14000096180, 0x14000096300, 0x140000e60e0)
	/home/ian/src/evil/lefthook/internal/config/load.go:147 +0x70
github.com/evilmartians/lefthook/internal/config.unmarshalConfigs(0x14000096180, 0x14000096300, 0x140000e60e0)
	/home/ian/src/evil/lefthook/internal/config/load.go:109 +0xc4
github.com/evilmartians/lefthook/internal/config.Load({0x10086d440, 0x100aaeda8}, {0x14000136280, 0x30})
	/home/ian/src/evil/lefthook/internal/config/load.go:38 +0x108
github.com/evilmartians/lefthook/internal/lefthook.(*Lefthook).Run(0x14000092040, {0x16f9735c5, 0xa}, {0x14000111310, 0x0, 0x0})
	/home/ian/src/evil/lefthook/internal/lefthook/run.go:48 +0xe4
github.com/evilmartians/lefthook/internal/lefthook.Run(0x1400012a1f8, {0x16f9735c5, 0xa}, {0x14000111310, 0x0, 0x0})
	/home/ian/src/evil/lefthook/internal/lefthook/run.go:39 +0x80
main.newRunCmd.func1(0x140001aec80, {0x14000111310, 0x1, 0x1})
	/home/ian/src/evil/lefthook/cmd/lefthook/run.go:18 +0x70
github.com/spf13/cobra.(*Command).execute(0x140001aec80, {0x140001112f0, 0x1, 0x1})
	/home/ian/go/pkg/mod/github.com/spf13/cobra@v1.2.1/command.go:856 +0x668
github.com/spf13/cobra.(*Command).ExecuteC(0x140001ae000)
	/home/ian/go/pkg/mod/github.com/spf13/cobra@v1.2.1/command.go:974 +0x410
github.com/spf13/cobra.(*Command).Execute(...)
	/home/ian/go/pkg/mod/github.com/spf13/cobra@v1.2.1/command.go:902
main.main()
	/home/ian/src/evil/lefthook/cmd/lefthook/main.go:12 +0x24
```

I've installed it via brew, rubygems, go, and by directly downloading the binary for the ARM macs.

I just downgraded to 0.8.0 and that version is working just fine for me. Happy to help debug if you need additional information. Otherwise, this is out of my wheelhouse to look into myself.

--- a/internal/config/command.go
+++ b/internal/config/command.go
@@ -72,7 +72,12 @@ func mergeCommands(base, extra *viper.Viper) (map[string]*Command, error) {
 	for key := range commandsOrigin.AllSettings() {
 		var replace commandRunReplace
 
-		if err := commandsOrigin.Sub(key).Unmarshal(&replace); err != nil {
+		substructure := commandsOrigin.Sub(key)
+		if substructure == nil {
+			continue
+		}
+
+		if err := substructure.Unmarshal(&replace); err != nil {
 			return nil, err
 		}
 

